diode-server/reconciler/ingestion_processor_internal_test.go

```diff
@@ -0,0 +1,406 @@
+package reconciler
+
+import (
+	"context"
+	"errors"
+	"log/slog"
+	"os"
+	"testing"
+
+	"github.com/redis/go-redis/v9"
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/protobuf/proto"
+
+	"github.com/netboxlabs/diode/diode-server/gen/diode/v1/diodepb"
+	"github.com/netboxlabs/diode/diode-server/netbox"
+	"github.com/netboxlabs/diode/diode-server/netboxdiodeplugin"
+	mnp "github.com/netboxlabs/diode/diode-server/netboxdiodeplugin/mocks"
+	"github.com/netboxlabs/diode/diode-server/reconciler/changeset"
+	mr "github.com/netboxlabs/diode/diode-server/reconciler/mocks"
+)
+
+func strPtr(s string) *string { return &s }
+func TestWriteJSON(t *testing.T) {
```

Comment:
nit: can we add line break between these functions, please?

Reply:
done